pupil-spa/src/app/check/check.component.spec.ts

```diff
@@ -227,7 +229,7 @@ describe('CheckComponent', () => {
     it('shows the question when the state is "Q<digit>"', () => {
       testStateChange('Q1', 'question', false);
     });
-    it('shows the submission pending screen when the state is "submission-pending"', () => {
+    xit('shows the submission pending screen when the state is "submission-pending"', () => {
```

Comment:
The tests are ignored because there are no server calls now — there is no way to log in and take the test as normal; the normal flow is through the survey.

The pending submission screen is ‘replaced’ with `ict-survey/preview-completed`/